DER/DerivativesContracts/CommoditiesContracts.rdf

```diff
@@ -136,7 +137,9 @@
 		<rdfs:subClassOf rdf:resource="&fibo-der-drc-comm;CommodityDerivative"/>
 		<rdfs:subClassOf rdf:resource="&fibo-der-drc-opt;VanillaOption"/>
 		<rdfs:label xml:lang="en">commodity option</rdfs:label>
-		<skos:definition xml:lang="en">option whose underlying asset is at least one commodity, commodity index, and/or commodity-specific contract</skos:definition>
+		<skos:definition xml:lang="en">option where the option buyer has the right to buy or sell specified commodities assets at a fixed price or formula, on or before a specified date</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Commodity assets may include at least one commodity, commodity index, and/or commodity-specific contract.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
not sure "commodity asset" is correct here - especially if you're talking about an index there is nothing owned hence no asset surely